docs/news.rst

```diff
@@ -288,6 +288,14 @@ Backward-incompatible changes
     :class:`~scrapy.http.Request` objects instead of arbitrary Python data
     structures.
 
+*   An additional `crawler` paramter has been added to the `__init__` method of
+    the `scrapy.core.scheduler.Scheduler` class. Custom scheduler subclassses 
+    don't accept binary parameters in their `__init__` method. This version of
```

Comment:
what does "binary" mean here?

Reply:
@kmike my bad, I meant to write arbitrary.